app/src/main/java/com/example/securityapplication/InputValidation.java

```diff
@@ -0,0 +1,26 @@
+package com.example.securityapplication;
+
+
+import android.content.Context;
+import android.support.design.widget.TextInputEditText;
+import android.support.design.widget.TextInputLayout;
+
+public class InputValidation {
+    private Context context;
+    public InputValidation(Context context) {
+        this.context = context;
+    }
+
+    public boolean is_Empty(TextInputEditText textInputEditText,
```

Comment:
Where is the code that is validating the length of mobile number and aadhar number? @WadekarIsha